reverse_engineering/databaseService/databaseService.js

```diff
@@ -7,7 +7,16 @@ const { logAuthTokenInfo } = require('../helpers/logInfo');
 const { getConnection } = require('./helpers/connection');
 const {
 	queryForRetrievingTheTablesSelectedByTheUser,
-} = require('../queries/queryForRetrievingTheTablesSelectedByTheUser');
+} = require('../queries/selectedTablesSubQuery/QueryForRetrievingTheTablesSelectedByTheUser');
+const {
+	PartitionsQueryForRetrievingTheTablesSelectedByTheUser,
```

Comment:
@WilhelmWesser Isn't naming inverted here as what you actually want is to retrieve the partition for the tables selected by the user?

`QueryForRetrievingPartitonsOfTablesSelectedByTheUser` is more correct but quite long :p

Reply:
Actually, it is quite tricky. In this query I need to get the tables that will be used in tables' partitions query

Reply:
I agree that it should be changed but in another way

Reply:
How do they differ from the ones selected by the user?

Reply:
The tables I am going to select are the ones that were selected by the user but we can not say that this class will represent the whole partitions query but only the tables subquery